versioned_docs/version-8.7/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -329,6 +329,6 @@ The following conventions apply to all attributes:
 
 <!--- TODO: insert link to C8 REST API guidelines --->
```

Comment:
❓ Does it really make sense to have this guide in 8.7 already? I believe this is only suited for 8.8, to be honest. This is one of things, as mentioned earlier, that we added to "next" already and now does not really make sense for "8.7 next" if you ask me. In the end, 8.7 now only is 8.6 with a couple of RPA/IDP/AI/Document handling on top. The docs should reflect this the same way, I think. Not saying that is simple to do, but we'll encounter more of these "not yet for 8.7" parts, I feat.

Reply:
I don't have a strong opinion here, but I'd prefer to handle this in a follow-up.